data/borderlines.json

```diff
@@ -127,6 +127,47 @@
         }
       ]
     },
+    {
+      "name": "Syangja",
+      "latitude": 28.0197,
+      "longitude": 83.8049,
+      "timezone": "Asia/Kathmandu",
+      "officialLanguage": "Nepali",
+      "currency": {
+        "name": "Nepalese Rupee",
+        "code": "NPR"
+      },
+      "landArea": {
+        "value": 1164,
+        "unit": "km²"
+      },
+      "elevation": {
+        "value": 1012,
```

Comment:
This still isn't correct
When I searched for elevation I found this
![image](https://github.com/Know-Nepal/borderlines/assets/60746512/46cc5197-2166-4e12-9d24-b7bb103266ba)